src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -610,3 +610,154 @@ public void Test()
     }
 }
 #endregion
+
+#region SomeFixtureWithExceptions
+[TestFixture]
+public class SomeBaseFixtureExceptionInSetUp
+{
+	private readonly bool exceptionInFixtureSetUp;
+	private readonly bool exceptionInSetUp;
+	public SomeBaseFixtureExceptionInSetUp(): this(false, true)
+	{		
+	}
+	public SomeBaseFixtureExceptionInSetUp(bool exceptionInFixtureSetUp, bool exceptionInSetUp)
+	{
+		this.exceptionInFixtureSetUp = exceptionInFixtureSetUp;
+		this.exceptionInSetUp = exceptionInSetUp;
+	}
+	[TestFixtureSetUp]
+	public void TestFixtureSetUp()
+	{
+		if (exceptionInFixtureSetUp)
+		{
+			NUnit.TestUtilities.SimpleEventRecorder.RegisterEvent("SomeBaseFixture.TestFixtureSetUp_Exception");
```

Comment:
I know it is used in a few places above, but this call is a bit long. At a minimum, you don't need the `NUnit.` portion, but maybe we should clean it up and add a using for `NUnit.TestUtilities`?